src/models/events.js

```diff
@@ -38,14 +42,18 @@ const events = function events() {
 
   function createTodoProcedure() {
     const todoValues = getFormValues();
-    const newTodo = createToDoItem(todoValues[0], todoValues[1], todoValues[2], todoValues[3], todoValues[4]);
-    const projectId = document.getElementById('projectId').innerHTML;
-    const ourStore = storage();
-    ourStore.addTodo(projectId, newTodo);
-    const closeModalButton = document.getElementById('closeModalButton');
-    closeModalButton.click();
-    document.getElementById('todoForm').reset();
-    dom().showTodos(projectId);
+    if (!todoValues.includes('')) {
+      const newTodo = createToDoItem(todoValues[0], todoValues[1], todoValues[2], todoValues[3], todoValues[4]);
+      const projectId = document.getElementById('projectId').innerHTML;
+      const ourStore = storage();
+      ourStore.addTodo(projectId, newTodo);
+      const closeModalButton = document.getElementById('closeModalButton');
+      closeModalButton.click();
+      document.getElementById('todoForm').reset();
+      dom().showTodos(projectId);
+    } else {
+      window.alert('You missed fill up one or more fields!');
```

Comment:
I trust you can do this 👇  before merging, `required` is already doing the job
`This should be production code, we shouldn't have `alert`, please remove the `alert`.`